# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/sharedmem_ipc_server.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/sharedmem_ipc_server.h
--- a/security/sandbox/chromium/sandbox/win/src/sharedmem_ipc_server.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/sharedmem_ipc_server.h	Wed Jun 12 11:10:48 2019 +0100
@@ -49,8 +49,10 @@
   // target_process_id: process id of the target process.
   // thread_provider: a thread provider object.
   // dispatcher: an object that can service IPC calls.
-  SharedMemIPCServer(HANDLE target_process, DWORD target_process_id,
-                     ThreadProvider* thread_provider, Dispatcher* dispatcher);
+  SharedMemIPCServer(HANDLE target_process,
+                     DWORD target_process_id,
+                     ThreadProvider* thread_provider,
+                     Dispatcher* dispatcher);
 
   ~SharedMemIPCServer();
 
@@ -65,14 +67,14 @@
   // When an event fires (IPC request). A thread from the ThreadProvider
   // will call this function. The context parameter should be the same as
   // provided when ThreadProvider::RegisterWait was called.
-  static void __stdcall ThreadPingEventReady(void* context,
-                                             unsigned char);
+  static void __stdcall ThreadPingEventReady(void* context, unsigned char);
 
   // Makes the client and server events. This function is called once
   // per channel.
   bool MakeEvents(base::win::ScopedHandle* server_ping,
                   base::win::ScopedHandle* server_pong,
-                  HANDLE* client_ping, HANDLE* client_pong);
+                  HANDLE* client_ping,
+                  HANDLE* client_pong);
 
   // A copy this structure is maintained per channel.
   // Note that a lot of the fields are just the same of what we have in the IPC
@@ -104,7 +106,8 @@
 
   // Looks for the appropriate handler for this IPC and invokes it.
   static bool InvokeCallback(const ServerControl* service_context,
-                             void* ipc_buffer, CrossCallReturn* call_result);
+                             void* ipc_buffer,
+                             CrossCallReturn* call_result);
 
   // Points to the shared memory channel control which lives at
   // the start of the shared section.